code/modules/mob/hear_say.dm

```diff
@@ -9,6 +9,8 @@
 			//Or someone snoring.  So we make it where they won't hear it.
 		return
 
+	message = capitalize(message)
```

Comment:
@Crazylemon64 It probably would have _functioned_
but dear god making all incoming messages automatically capitialized screams "WHY"